template/web/app/paymaster-bundler/api/paymasterProxy/route.ts

```diff
@@ -0,0 +1,39 @@
+import { NextRequest, NextResponse } from 'next/server';
+import { UserOperation } from 'permissionless';
+import { paymasterClient } from '../../../../../web/src/components/SmartWallets/PaymasterClient';
+
+type PaymasterRequest = {
+  method: string;
+  params: [UserOperation<'v0.6'>, string, string];
+};
+
```

Comment:
Reflecting on the name of the file `/api/paymasterProxy`, it means the api will be CamelCase, which I don't think it's a good practice.

should this be `paymaster-proxy`?

Reply:
Hmm perhaps, other apis in the repo follow camelCase though https://github.com/coinbase/build-onchain-apps/blob/paul/add-paymaster-proxy/template/web/app/api/chain/currentBlockNumber/route.ts

Reply:
I feel think `paymaster-proxy` is better. I can refactor all of the routes in this repo as a fast follow

Reply:
nice